src-care/topkg_care.mli

```diff
@@ -486,6 +486,7 @@ module Delegate : sig
   (** {2 Helpers} *)
 
   val publish_in_git_branch :
+    ?remote:string ->
```

Comment:
need to add `and pushes the branch to [remote] (defaults to ["origin"])` to the doc string.